frontend/src/pages/ActivityReport/index.js

```diff
@@ -82,6 +83,8 @@ function ActivityReport({ match, user, location }) {
   const [initialLastUpdated, updateInitialLastUpdated] = useState();
   const reportId = useRef();
 
+  const regionIdToUse = region || getRegionWithReadWrite(user);
```

Comment:
This or statement will never evaluate the RHS since region is explicitly set up above. This will change once we move away from setting the region explicitly